### PR TITLE
add ConfidentialBox UseSbieDeskHack docs

### PR DESCRIPTION
**Pull request by wwxiaoqi**
I have added the relevant documentation for `ConfidentialBox` `UseSbieDeskHack`.

### CODE REVIEW

File: Content/UseSbieDeskHack.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+A desktop object solution that is now enabled by default for all processes.
+
+
+**Technical Details**
+
+This is a desktop object solution that is used for all processes. 
+
+It was initially implemented to address the issue of infinite callback problems caused by delayed loading (the infinite recursion problem has been resolved in version 5.43).
```
Comment (isaak654) [inline Content/UseSbieDeskHack.md]: > (the infinite recursion problem has been resolved in version 5.43).

I know you took this info from [here](https://github.com/sandboxie-plus/Sandboxie/commit/43d61c50bc0420790f13dc4b79deb7a2b27f717d#diff-55d44deb107f22cc4864816fe46df3f3e64fa82aa00af5fd69d270cbe70d36f7), but could you point me to the lines that introduced the recursion fix? The note you refer to was written some time after the fix, so I would like to be sure it is accurate.

File: Content/UseSbieDeskHack.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+**Technical Details**
+
+This is a desktop object solution that is used for all processes. 
+
+It was initially implemented to address the issue of infinite callback problems caused by delayed loading (the infinite recursion problem has been resolved in version 5.43). 
+
+It is now enabled by default. This allows Electron applications to run without the need to set the 'SpecialImage=chrome,program.exe' option.
+
+
+You can find it in the [Sandboxie Control](SandboxieControl.md) Setting now:
```
Comment (isaak654) [inline Content/UseSbieDeskHack.md]:
```suggestion
Related Sandboxie Plus setting:
```

File: Content/AllPages.md [hunk 3]
```diff
@@ -642,6 +644,8 @@
 
 [User Accounts Settings](UserAccountsSettings.md)
 
+[Use SbieDesk Hack](UseSbieDeskHack.md)
```
Comment (isaak654) [inline Content/AllPages.md]: This should be moved above `[Use Security Mode](UseSecurityMode.md)`

File: Content/AllPages.md [hunk 4]
```diff
@@ -82,6 +82,8 @@
 
 [Copy Limit Silent](CopyLimitSilent.md)
 
+[Confidential Box](ConfidentialBox.md)
```
Comment (isaak654) [inline Content/AllPages.md]: This should be moved above `[Config Level](ConfigLevel.md)`

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Content/UseSbieDeskHack.md (hunk 1 above)
Comment (wwxiaoqi) [reply Content/UseSbieDeskHack.md]: At that time, I examined the relevant code and ultimately located it in the `Build 0.4.0 / 5.43` submission of version `v0.3...v0.4.0` on [Github/Sandboxie/v0.3...v0.4.0#diff](https://github.com/sandboxie-plus/Sandboxie/compare/v0.3...v0.4.0#diff-60bf43005a5ae1c852835126db27eabf6130844c67f51abad8c8d471c4c791e8).

File: Content/UseSbieDeskHack.md (hunk 2 above)
Comment (wwxiaoqi) [reply Content/UseSbieDeskHack.md]: It has already been modified in the latest version.

File: Content/UseSbieDeskHack.md (hunk 1 above)
Comment (isaak654) [inline Content/UseSbieDeskHack.md]:
```suggestion
It was initially implemented to address the issue of infinite callback problems caused by delayed loading (the infinite recursion problem has been resolved in version 0.4.0 / 5.43).
```

File: Content/UseSbieDeskHack.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+   [DefaultBox]
+   UseSbieDeskHack=y
+```
+
+A desktop object solution that is now enabled by default for all processes.
+
+
+**Technical Details**
+
+This is a desktop object solution that is used for all processes.
```
Comment (isaak654) [inline Content/UseSbieDeskHack.md]:
```suggestion
This is a desktop object solution that is used for all processes.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Comment by isaak654** [comment]
This PR cannot be approved without considering the following changes: https://github.com/sandboxie-plus/sandboxie-docs/pull/145#pullrequestreview-1797930184

**Review by isaak654 — COMMENTED** [review]
This PR cannot be approved without considering the following changes: https://github.com/sandboxie-plus/sandboxie-docs/pull/145#pullrequestreview-1797930184

**Comment by wwxiaoqi** [comment]
> This PR cannot be approved without considering the following changes: [#145 (review)](https://github.com/sandboxie-plus/sandboxie-docs/pull/145#pullrequestreview-1797930184)

Sorry, my browser didn't refresh.

File: Content/ConfidentialBox.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+   [DefaultBox]
+   ConfidentialBox=y
+```
+
+Use the 'ConfidentialBox=y' option to prevent the host process from reading access to the isolated process.
+
+
+**Technical Details**
+
+Implement encryption using [DiskCryptor](https://diskcryptor.org/) and use the 'ConfidentialBox=y' option to let SbieDrv block non-sandboxed processes from accessing encrypted sandbox folders.
```
Comment (isaak654) [inline Content/ConfidentialBox.md]:
```suggestion
For more information, see [Box Encryption](../PlusContent/BoxEncryption.md) and [Box Preset Comparison](../PlusContent/box-preset-comparison.md).
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Content/UseSbieDeskHack.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+It was initially implemented to address the issue of infinite callback problems caused by delayed loading (the infinite recursion problem has been resolved in version 0.4.0 / 5.43).
+
+It is now enabled by default. This allows Electron applications to run without the need to set the 'SpecialImage=chrome,program.exe' option.
+
+
+Related Sandboxie Plus setting:
+
+Sandbox Options > Various Options > Compatibility > Use desktop object workaround for all processes
+
+
```
Comment (isaak654) [inline Content/UseSbieDeskHack.md]:
```suggestion
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — APPROVED** [review]

**Pull request merged**